aws/resource_aws_dms_endpoint_test.go

```diff
@@ -95,7 +100,58 @@ func TestAccAwsDmsEndpoint_S3(t *testing.T) {
 		},
 	})
 }
+func TestAccAwsDmsEndpoint_S3_Parquet(t *testing.T) {
+	resourceName := "aws_dms_endpoint.dms_endpoint"
+	randId := acctest.RandString(8) + "-s3"
 
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: dmsEndpointDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: dmsEndpointS3Config(randId),
+				Check: resource.ComposeTestCheckFunc(
+					checkDmsEndpointExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "s3_settings.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "s3_settings.0.bucket_folder", ""),
+					resource.TestCheckResourceAttr(resourceName, "s3_settings.0.bucket_name", "bucket_name"),
+					resource.TestCheckResourceAttr(resourceName, "s3_settings.0.compression_type", "NONE"),
+					resource.TestCheckNoResourceAttr(resourceName, "s3_settings.0.enable_statistics"),
+					resource.TestCheckNoResourceAttr(resourceName, "s3_settings.0.timestamp_column_name"),
```

Comment:
I don't know how to get this test to pass, even though the response object clearly indicates that the attribute is not set.